packages/ra-core/src/controller/field/ReferenceFieldController.tsx

```diff
@@ -1,10 +1,6 @@
-import { Component, ReactNode } from 'react';
-import { connect } from 'react-redux';
-import get from 'lodash/get';
-
-import { crudGetManyAccumulate as crudGetManyAccumulateAction } from '../../actions';
-import { linkToRecord } from '../../util';
-import { Record, Dispatch } from '../../types';
+import { SFC, ReactNode, ReactElement } from 'react';
```

Comment:
I think they renamed `SFC` into `FunctionalComponent` in the latest typings. Probably not a good idea to upgrade the typings in this PR though

Reply:
For reference: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30364